AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -620,6 +620,12 @@ rt.com###offers
 !+ NOT_OPTIMIZED
 mdpr.jp###app-footer-banner
 !
+!#if (android)
```

Comment:
What is `!#if (` **android** `)`?
https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#what-constants-do-we-declare